testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/util/DeltaLakeTestUtils.java

```diff
@@ -29,14 +29,15 @@ private DeltaLakeTestUtils() {}
 
     public static String getDatabricksRuntimeVersion()
     {
-        QueryResult result = onDelta().executeQuery("SELECT java_method('java.lang.System', 'getenv', 'DATABRICKS_RUNTIME_VERSION')");
-        return firstNonNull((String) result.row(0).get(0), "unknown");
+        String sql = "SELECT java_method('java.lang.System', 'getenv', 'DATABRICKS_RUNTIME_VERSION')";
+        return firstNonNull((String) onDelta().executeQuery(sql).getOnlyValue(), "unknown");
     }
 
     public static String getColumnCommentOnTrino(String schemaName, String tableName, String columnName)
     {
-        QueryResult result = onTrino().executeQuery("SELECT comment FROM information_schema.columns WHERE table_schema = '" + schemaName + "' AND table_name = '" + tableName + "' AND column_name = '" + columnName + "'");
```

Comment:
Generally i prefer to inline queries, since these variables do not add to readability a lot.
i am fine either way here, since it's an isolated method

also, the use of concatenation over `format` is intentional
For some reason, in case of concatenation, the IntelliJ can syntax highlight the query text, and sadly it cannot do this when `format` is used.